### ISSUE TITLE
Linux Mint Debian Edition

### ISSUE THREAD

**Issue by lfromanini** [wimpysworld/deb-get#399]
Please, make it available to LMDE. Currently, the following error is raised:

```bash
curl -sL https://raw.githubusercontent.com/wimpysworld/deb-get/main/deb-get | sudo -E bash -s install deb-get
  [!] ERROR! Linux Mint Elsie is not supported because it is not derived from a supported Debian or Ubuntu release.
```

**Comment by Xmetalfanx** [comment]
I am seeing the same thing in an LMDE 5 VM i just installed and I'd login there to give the required info but seeing a few merges... I feel this issue may be related to https://github.com/wimpysworld/deb-get/issues/384  and fixed on https://github.com/wimpysworld/deb-get/commit/fe4cb91c60535c438af685894a4093e58eb241d4 or related commits 

the installed version I have when seeing this is 0.3.3 of deb-get

**Comment by flexiondotorg** [comment]
Closed via af676d053f96e0dbcde3cbd3944f195bd63fed6d
